common/src/main/java/com/microsoft/identity/common/internal/platform/DevicePopManager.java

```diff
@@ -1109,6 +1141,16 @@ private KeyPair generateNewRsaKeyPair(@NonNull final Context context,
         );
     }
 
+    private static boolean isStrongBoxUnavailableException(@NonNull final Throwable t) {
```

Comment:
Should we rename the method to `isStrongBoxUnavailable` ? seems like that's the intent? it just uses the exception to figure that out

Reply:
I'll go ahead and leave this as-is, as this method's check accepts the _error_ as the argument, rather than being a 'general purpose' API for discovering whether/not StrongBox is supported